src/components/RichTextInput/RichTextInput.module.scss

```
@@ -0,0 +1,125 @@
+﻿@import "../../styles/mixins";
+
+.rich-text {
+  display: block;
+
+  .rich-text__message {
+    @include bravo;
+    font-style: italic;
+    font-weight: var(--font-weight-book);
+    margin-top: 0.35rem;
+    color: var(--neutral-60);
+  }
+
+  .rich-text__label {
+    @include bravo;
+    color: var(--neutral-70);
+    padding-bottom: var(--spacer-three-xs);
+    transition: color var(--transition-regular);
+  }
+
+  .rich-text__suspense-loading {
+    min-height: 140px;
+  }
+
+  :global {
+    .jodit-container {
+
+      &:not(.jodit_inline) {
+        border-radius: 4px;
+        border-color: var(--neutral-40);
+      }
+
+      .jodit-toolbar__box,
+      .jodit-ui-separator {
+        border-color: var(--neutral-40);
+      }
+
+      .jodit-toolbar-button {
+        button svg {
+          fill: var(--brand-aubergine);
+        }
+      }
+
+      .jodit-workplace {
+        color: var(--brand-aubergine);
```

Please increase the font-size to 14px
```suggestion
        color: var(--brand-aubergine);
        font-size: 14px;
```